bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/fields/_buttons.html.erb

```diff
@@ -1,23 +1,31 @@
 <%
 stimulus_controller = 'fields--button-toggle'
 form ||= current_fields_form
+
+# Initialize options.
 html_options ||= {}
-html_options[:id] ||= form.field_id(method)
-multiple ||= false
+options ||= {}
 other_options ||= {}
-options ||= options_for(form, method)
+button_field_options ||= options_for(form, method)
+
+# For backwards compatibility.
+options.merge!(html_options)if options.is_a?(Hash)
+
+options[:id] ||= form.field_id(method)
+
+multiple ||= false
 %>
 
-<% content = render 'shared/fields/field', form: form, method: method, options: html_options, other_options: other_options do %>
+<% content = render 'shared/fields/field', form: form, method: method, options: options, other_options: other_options do %>
```

Comment:
What if we pass the id in here instead? It seems like it's potentially difficult that every other use of `options` has to remember to do `options.except(:id)`.

```suggestion
<% content = render 'shared/fields/field', form: form, method: method, options: options.with_defaults(id: form.field_id(method)), other_options: other_options do %>
```

Reply:
It looks like we have this (`options[:id] = form.field_id(method)`) in a lot of different places. I'll update this for `_buttons.html.erb`, but I think I'll make the changes for the different partials in another PR.